inspec/a2-iam-no-legacy-integration/controls/chef_managed_access.rb

```diff
@@ -4,146 +4,120 @@
 
 require_relative '../../constants'
 
-title 'IAM v2.1 role access API integration tests'
+title 'iam chef-managed policy access integration tests'
 
-control 'iam-v2p1-roles-1' do
-  title 'v2p1-only access'
-  desc 'role-based access for editor and viewer when v1 policies are purged'
+control 'iam-chef-managed-access-1' do
+  title 'iam chef-managed policy access'
+  desc 'role-based access for chef-managed policies on a fresh system with no legacy policies'
 
-  VIEWER_USERNAME = 'viewer'
-  VIEWER_ROLE = 'viewer'
-  VIEWER_ROLE_ACCESS_POLICY_ID = 'viewer-inspec-role-access-test'
+  VIEWER_USER_ID = "inspec-viewer-#{TIMESTAMP}"
+  EDITOR_USER_ID = "inspec-editor-#{TIMESTAMP}"
+  PROJECT_OWNER_USER_ID = "inspec-project-owner-#{TIMESTAMP}"
 
-  EDITOR_USERNAME = 'editor'
-  EDITOR_ROLE = 'editor'
-  EDITOR_ROLE_ACCESS_POLICY_ID = 'editor-inspec-role-access-test'
+  ALL_ROLES = [ADMIN_USER_ID, VIEWER_USER_ID, EDITOR_USER_ID, PROJECT_OWNER_USER_ID]
```

Comment:
Yes, in a sense these are "roles" but that confuses them with "Roles" (our specific objects meaning a set of actions). So perhaps `USERS_WITH_ALL_ROLES`...?